src/main/scala/org/apache/spark/sql/rikai/model/ModelResolver.scala

```diff
@@ -44,41 +82,64 @@ object ModelResolver {
       Python.execute(
         s"""from pyspark.serializers import CloudPickleSerializer;
```

Comment:
nit: ident

Reply:
hmm, this is the result of `scalafmt`.  I tried to change it a bit, and `scalafmt` complains